charts/opentelemetry-operator/Chart.yaml

```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: dmitryax
   - name: TylerHelmuth
 icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
-appVersion: 0.56.0
+appVersion: 0.58.0
```

Comment:
Should this be 0.59.0?

Reply:
nope https://github.com/open-telemetry/opentelemetry-operator/releases

Reply:
[This](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/352/files#diff-56c52957e9b80d7dcb9b9771e329d7dc4f1a3ee8fb22acdcb3a75c1c92e75c18R66) is actually an workaround for that issue. Added a comment ao we remove it once 0.59 gets released.